run_test_docker.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+docker run -it --entrypoint '/home/jovyan/run_tests.sh' eu.gcr.io/unit8-product/u8timeseries:latest
```

Comment:
Is this used somewhere in github action?

Reply:
No, I used this to test the same behaviour locally. Should I remove it?